src/components/form-draft/testing.vue

```diff
@@ -147,7 +166,15 @@ export default {
     ],
     // This text will be shown in ODK Collect when testing a Draft Form. {name}
     // is the title of the Draft Form.
-    "collectProjectName": "[Draft] {name}"
+    "collectProjectName": "[Draft] {name}",
+    "datasetsPreview": {
+      "title": "Datasets Preview",
+      "body": [
+        "This form can add entities to a dataset.",
+        "In this preview release of Datasets, you must publish your Form to see the Dataset update. In a future release of Central, you will be able to test Dataset functionality in a draft state.",
+        "For now, we recommend publishing your Dataset-related Forms, and not approving any submissions (and therefore not create any unwanted entities) until you are sure the Form is ready."
```

Comment:
I think this string has changed. We often iterate on text after the screenshot/mock-up is created: the body of the release criteria has the most up-to-date text. (That's true in general: where it differs from the screenshot, it's the body/text of the release criteria that reflects the latest truth.)

Reply:
Recent text in release criteria is bit confusing, I have left a question there

Reply:
As discussed in the weekly meeting, we are keeping this text as it is and putting document link on the manage attachments (media) page